tox.ini

```diff
@@ -1,9 +1,9 @@
 [tox]
 envlist =
 	py35-django{21,22}
-	py36-django{21,22,30,31}
-	py37-django{21,22,30,31}
-	py38-django{21,22,30,31}
+	py36-django{21,22,30,31,master}
+	py37-django{21,22,30,31,master}
+	py38-django{21,22,30,31,master}
 	docs
```

Comment:
this has been removed on purpose from declared environments to allow to run `tox` without any failure